### PR TITLE
Feature/map diagram

### PR DESCRIPTION
**Pull request by vlach1989**

### CODE REVIEW

File: src/ReactLeafletMap/layers/VectorLayer/index.js [hunk 1]
```diff
@@ -25,31 +25,38 @@ class VectorLayer extends React.PureComponent {
...
+        if (!style.fill) {
+            finalStyle.fillColor = null;
+            finalStyle.fillOpacity = 0;
+        }
+
+        if (!style.outlineColor || !style.outlineWidth) {
+            finalStyle.color = null;
+            finalStyle.opacity = 0;
+            finalStyle.weight = 0;
+        }
```
Comment (tmdmc) [inline src/ReactLeafletMap/layers/VectorLayer/index.js:59]: Are these general defaults, or framework-specific?

File: src/ReactLeafletMap/layers/DiagramLayer/index.js [hunk 2]
```diff
@@ -16,19 +16,30 @@ class DiagramLayer extends VectorLayer {
         return this.getDiagramLeafletStyle(feature, defaultStyleObject);
     }
 
-    getDiagramAccentedStyle(feature, defaultStyleObject, accentedStyleObject) {
-        const style = {...defaultStyleObject, ...accentedStyleObject};
+    getDiagramAccentedStyle(feature, defaultStyleObject, styleDefinition) {
```
Comment (tmdmc) [inline src/ReactLeafletMap/layers/DiagramLayer/index.js:19]: What's the difference between styleObject and styleDefinition?

File: src/ReactLeafletMap/layers/VectorLayer/index.js (hunk 1 above)
Comment (vlach1989) [reply src/ReactLeafletMap/layers/VectorLayer/index.js:59]: Framework-specific, necessary to draw (or rather not to draw) desired properties and reset framework defaults

File: src/ReactLeafletMap/layers/DiagramLayer/index.js (hunk 2 above)
Comment (vlach1989) [reply src/ReactLeafletMap/layers/DiagramLayer/index.js:19]: Better naming would be defaultStyleObject and accentedStyleObject - both objects are in panther style notation

**Pull request merged**